src/inline-plugin.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+    tinymce.PluginManager.add('inline_tinymce_plugin', function(editor, url) {
```

Comment:
You might want to eventually split this up into separate objects and then compose those smaller objects into one large object at the end of the file.

That way, you won't run into the issue of nesting hell and all the confusion that comes with it.